backend/src/main.rs

```diff
@@ -316,6 +316,40 @@ async fn cache_hub_scripts(file_path: Option<String>) -> anyhow::Result<()> {
     Ok(())
 }
 
+fn print_help() {
+	println!("Windmill - a fast, open-source workflow engine and job runner.");
+	println!();
+	println!("Usage:");
+	println!("  windmill [SUBCOMMAND]");
+	println!();
+	println!("Subcommands:");
+	println!("  help                 Show this help information and exit");
+	println!("  version              Show Windmill version and exit");
+	println!("  cache [hubPaths.json]  Pre-cache hub scripts (default: ./hubPaths.json)");
+	println!();
+	println!("Configuration via environment variables (name = default):");
+	println!("  DATABASE_URL = <required>   PostgreSQL connection string");
+	println!("  NUM_WORKERS = {}            Number of workers (standalone/worker modes)", DEFAULT_NUM_WORKERS);
+	println!("  DISABLE_SERVER = false      Disable HTTP server when true");
+	println!("  SERVER_BIND_ADDR = {}       IP to bind the server to", DEFAULT_SERVER_BIND_ADDR);
+	println!("  PORT = {}                   Server port (in server/indexer/MCP modes)", DEFAULT_PORT);
+	println!("  RUST_LOG = info             Rust logging level (e.g., trace, debug, info)");
```

Comment:
some env variables are present (SKIP_MIGRATION, DISABLE_SERVER) but not very important while others are missing (MODE=worker|agent|server), JWT_AGENT_TOKEN, WORKER_GROUP